ovn_k8s/modes/overlay.py

```diff
@@ -310,6 +304,9 @@ def add_endpoint(self, event):
                 response_json = kubernetes.get_service(
                                                    variables.K8S_API_SERVER,
                                                    namespace, service_name)
+            except exceptions.NotFound:
+                vlog.dbg("No service found for endpoint %s " % service_name)
```

Comment:
I have been manually running flake8 on each file. I wonder whether there is a easy way to integrate flake8 such that we can call it on the entire repo.